fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -275,6 +280,15 @@ public RecordMetaDataProvider getMetaDataProvider() {
         return metaDataProvider;
     }
 
+    /**
+     * Get the local meta-data file descriptor.
```

Comment:
This should probably include a link to some other Javadoc that explains what the local meta-data file descriptor is useful for.

Reply:
sure.